crates/fluvio-connector/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "fluvio-connector"
+version = "0.0.0"
+edition = "2018"
+
```

Comment:
need to other metadata including whether this should be published. look at other crates